### PR TITLE
Arrays.fill vs Arrays.setAll methods

### PR DESCRIPTION
**Pull request by WiktoriaMajchrzak**
Create CompletingArray class to simply notice the difference between methods
Create CompletingArrayWithBenchmark class to check the runtime differences
Create CompletingArrayTest

### CODE REVIEW

**Comment by WiktoriaMajchrzak** [comment]
In answer to the question, which method (Arrays.fill or Arrays.setAll) would be better to fill an array with just numbers 1.

The SetAll method requires a function as a  second parameter and calls it for each function element. So it makes sense when the values ​​are to be different. In case you want the same it is better to use the fill method, which in this case will be much faster. setAll uses the functional interface to call its method, and will therefore run slower. So when I typed Arrays.setAll(arr, (index) -> 1), IntelliJ suggests using a more efficient function in this case, which is fill.

They both seem to have O(n) complexity in that case(with time and space), but this complexity for setAll can vary depending on the function provided, and it will looks like O(n * complexity of functions) for time, and O(n * complexity of functions + n) for space. Am i correct? Complexity is still a bit of a complicated topic for me, so I would love to know what the correct answer is if I gave a wrong one :)

File: src/main/java/com/epam/prejap/ess/helpers/CompletingArray.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
+package com.epam.prejap.ess.helpers;
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/CompletingArray.java:1]: Please enclose your solution int the additional package (inside the main one) with the name indicating the solved problem.

Comment (WiktoriaMajchrzak) [reply src/main/java/com/epam/prejap/ess/helpers/CompletingArray.java]: The packages have been added

**Comment by marcin-ogorzalek-EPAM** [comment]
Please resolve conflict in `pom.xml` and  bump up the application's version

File: src/main/java/com/epam/prejap/ess/helpers/ArraysFillVsSetAll/CompletingArray.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
+package com.epam.prejap.ess.helpers.ArraysFillVsSetAll;
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/ArraysFillVsSetAll/CompletingArray.java:1]: In package's name we rather avoiding camel or mixed case (by convention). Please change it to all lower case letters

Comment (WiktoriaMajchrzak) [reply src/main/java/com/epam/prejap/ess/helpers/ArraysFillVsSetAll/CompletingArray.java]: The name has been changed

+ Conflicts was resolved
+ Bump pom version

**Pull request merged**